Formula/f/flang.rb

```diff
@@ -0,0 +1,55 @@
+class Flang < Formula
+  desc "LLVM Fortran Frontend"
+  homepage "https://flang.llvm.org/"
+  url "https://github.com/llvm/llvm-project/releases/download/llvmorg-19.1.0/llvm-project-19.1.0.src.tar.xz"
+  sha256 "5042522b49945bc560ff9206f25fb87980a9b89b914193ca00d961511ff0673c"
+  # The LLVM Project is under the Apache License v2.0 with LLVM Exceptions
+  license "Apache-2.0" => { with: "LLVM-exception" }
+  head "https://github.com/llvm/llvm-project.git", branch: "main"
+
+  livecheck do
+    url :stable
+    regex(/^llvmorg[._-]v?(\d+(?:\.\d+)+)$/i)
+  end
```

Comment:
Should this be synchronized to `llvm` or at least livechecked to `formula "llvm"` given it links to static MLIR libraries? Otherwise, we can end up build with older MLIR libs.